recipe/meta.yaml

```
@@ -44,27 +37,34 @@ requirements:
     - py_stringsimjoin >=0.3.0
     - cloudpickle >=0.2.1
     - pyparsing >=2.1.4
-    - scikit-learn >=0.18,<0.22.0a0
-    - xgboost  # [not win]
-    - pyqt
+    - scikit-learn >=0.22
     - requests
-    - ipython  # ==5.6
+    - ipython >=5.6
     - matplotlib-base >=2.2.4
 
 test:
+  source_files:
+    - py_entitymatching/tests
   imports:
     - py_entitymatching
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    # skip test_null_cases, marked @unittest.skip("Not a test")
+    - pytest -v py_entitymatching/tests -k "not test_null_cases"
```

```suggestion
    - python -m unittest -v
```
I wonder if this would skip the test automatically. That is how upstream runs tests https://github.com/anhaidgroup/py_entitymatching/blob/v0.4.2/.github/workflows/testing.yml#L37C9-L37C30

---

Well, it doesn't _fail_:

```
+ python -m unittest -v

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```